qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -286,6 +288,9 @@ def run(
                 "'use_measure_esp' is unset or set to 'False'."
             )
 
+        if sim_method is None:
```

Comment:
```suggestion
        if not self.configuration().simulator:
```
Not all simulators have `simulator_method` in its config (e.g. `ibmq_qasm_simulator`).